source/docs/one-command-deployment.md

```diff
@@ -254,6 +254,28 @@ The sample app for `hexo` is deployed at [https://hexo.21yunbox.com/](https://he
 
 For more detail, follow this guide at [https://www.21yunbox.com/docs/#/deploy-hexo](https://www.21yunbox.com/docs/#/deploy-hexo).
 
+## Bip
+
+[Bip](https://bip.sh) provides zero downtime deployment, a global CDN, SSL, unlimited bandwidth and more for static websites.
```

Comment:
It appears that `Bip` is not a free service. IMHO it should be mentioned in the documents before `bip init`.

Reply:
@SukkaW sure. I've updated the docs to note right upfront that it's a commercial service, and have included the pricing structure (pay as you go, per domain). Let me know if that's OK.

Reply:
LGTM!